keyform/views.py

```diff
@@ -25,11 +27,55 @@ def get_queryset(self):
 
 class RequestView(LoginRequiredMixin, UpdateView):
     model = Request
+    success_url = reverse_lazy('home')
     template_name = "keyform/request.html"
     form_class = EditForm
 
     def get_success_url(self):
-        return reverse('home')
+        return reverse_lazy('home')
+
+class ContactView(LoginRequiredMixin, TemplateView):
+    template_name = "keyform/contact.html"
+
+    def get_context_data(self):
+        context = super(ContactView, self).get_context_data()
+        context["buildings"] = Building.objects.all()
+        return context
+
+    def post(self, request, *args, **kwargs):
+
+        if request.user.has_perm('keyform.delete_user'):
+            pk = request.POST['pk']
+            Contact.objects.filter(pk=pk).delete();
```

Comment:
Should we actually delete it or should we add a flag and just never show it. Additionally we could change the default [manager](https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset) for contact objects to not show the deleted ones.